rpcs3/Emu/RSX/VK/vkutils/device.cpp

```diff
@@ -183,9 +183,16 @@ namespace vk
 
 	driver_vendor physical_device::get_driver_vendor() const
 	{
+#ifdef __APPLE__
+		// moltenVK currently returns DRIVER_ID_MOLTENVK (0).
```

Comment:
FTR https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VkDriverId.html
MoltenVK is ID 14, ID 0 means the driverId feature is not supported. Usually I'd consider adding an assert just to be sure, but I don't think anyone cares.